state/store.go

```diff
@@ -36,10 +36,47 @@ func calcABCIResponsesKey(height int64) []byte {
 	return []byte(fmt.Sprintf("abciResponsesKey:%v", height))
 }
 
+//----------------------
+
+type StoreI interface {
```

Comment:
It would be awesome if these methods returned errors too. But we can postpone for later if you'd rather keep this PR as a simple interface/struct migration.

Reply:
I'll do this. Better to do it now while this change is being made.